LAMMPS/build_lammps_15Oct2020_gcc930.md

```diff
@@ -0,0 +1,77 @@
+Building LAMMPS on ARCHER2 (GCC 9.3.0)
+===================================================
+
+These instructions are for building LAMMPS on ARCHER2 using the GNU compilers
+
+Download LAMMPS
+---------------
+
+Clone the latest stable version of LAMMPS from the GitHub repository:
+
+   `git clone -b stable https://github.com/lammps/lammps.git lammps-20200713`
+
+Setup your environment
+----------------------
+
+Load the correct modules:
+
+   ```bash
+   module switch PrgEnv-cray/7.0.0 PrgEnv-gnu/7.0.0
```

Comment:
This cannot be right for ARCHER2 - surely it is `module restore PrgEnv-gnu`?

Reply:
Good spot -- I'll change that (also changed it in the Quantum Esspresso instructions)